django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -64,13 +66,82 @@ def get_random_search_term(target):
     return search_term
 
 
+class ListViewTest(ABC):
```

Comment:
Is this abstract class used anywhere? I don't see it referenced/inherited by other classes?

Reply:
this is a good point! I saw several "TODO: maybe write a class to test pagination in all list views" comments as I was working through the tests. It seems this was an effort to write such a class, but it never got plugged into anything. Let me do some experimenting and see whether this can be put to good use.

Reply:
spent a while experimenting with this, and it seems like more trouble than it's worth. I removed it for now, and I'll open a `lowpriority` issue in case someone wants to come back to it.